save.py

```diff
@@ -195,6 +196,22 @@ def rare_chance_as_str(self) -> tuple[str, str, str]:
     def upcoming_boss(self) -> str:
         return self._data["boss"]
 
+    @property
+    def bottled_cards(self) -> str:
+        cards = []
+        if self._data("bottled_flame"):
+            cards.append("🔥 " + self._get_card_string(self._data["bottled_flame"], self._data["bottled_flame_upgrade"]))
```

Comment:
I would rather use the emoji names than putting the actual emoji in (mostly as a matter of personal preference). The way to do this is the following (capital letters necessary):

🔥 = `"\N{FIRE}"` 
⚡ = `"\N{HIGH VOLTAGE SIGN}"`
🌪️ = `"\N{CLOUD WITH TORNADO}"`